### DISCUSSION TITLE
Posting Category Children as Outline on Page

### DISCUSSION THREAD

**Discussion by mweth** [outline/outline#5370] [General]
I'm sure there is an easier way to ask this question, but am not sure of the nomenclature of the page elements used.

Each collection has categories and categories can have sub pages.  Each category has a link at the bottom for Documents and Backlinks.

I would like to have a dynamically updated version of the Documents layout in hierarchical format to post at the top of the article as a table of contents to the child content contained in the category.  Is there already a shortcode or method to do this?  I have posted a text example to clarify my question.

![image](https://github.com/outline/outline/assets/20465873/b16d8aeb-07a3-49bd-86a2-d9faeeeef080)

**Comment by tommoor** [comment]
Hi @mweth – this already exists and is hidden under this icon in your screenshot…

![image](https://github.com/outline/outline/assets/380914/964f9af7-422d-45b4-afe0-deb7b1f244f1)
